### PR TITLE
add giddy recipe

### PR DESCRIPTION
**Pull request by knaaptime**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by knaaptime** [comment]
other maintainers of this recipe have indicated their approval here https://github.com/pysal/giddy/issues/89

File: recipes/giddy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  sha256: 6c284856579ab95ff6a63b6a7c838ab0a48c8669745ea16f8428a012abedb750
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >= 3
```
Comment (weikang9009) [inline recipes/giddy/meta.yaml]: giddy has not been tested against python 3.7 
[only 3.5 and 3.6](https://github.com/pysal/giddy/blob/master/.travis.yml#L6) at the moment
but I assume it is OK not to specify it here?

Comment (xhochy) [reply recipes/giddy/meta.yaml]: Couldn't you just test it? As it is a `noarch: python` package it should work and we're safe here with `python >= 3`.

Comment (knaaptime) [reply recipes/giddy/meta.yaml]: sounds like we're ok on this?

File: recipes/giddy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - giddy
+
+about:
+  home: http://github.com/pysal/giddy
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'GeospatIal Distribution DYnamics (giddy) in PySAL'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (chrisburr) [inline recipes/giddy/meta.yaml]:
```suggestion
```

File: recipes/giddy/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/giddy/meta.yaml]: The space between the operator and the number needs to be removed i.e `python >=3` (The space before `>=` is important)
```suggestion
    - python >=3
```
```suggestion
    - python >= 3
```

File: recipes/giddy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >= 3
+    - pip
+  run:
+    - python >= 3
```
Comment (chrisburr) [inline recipes/giddy/meta.yaml]: Same here:
```suggestion
    - python >=3
```

Comment (chrisburr) [reply recipes/giddy/meta.yaml]: And the others need a space adding to match this pattern. (Building can work but other parts of conda-build which are used for making the feedstock will fail if its not formatted correctly)

Comment (knaaptime) [reply recipes/giddy/meta.yaml]: cool, thank you! should be fixed now

**Pull request merged**